### DISCUSSION TITLE
Data

### DISCUSSION THREAD

**Discussion by amberwnaushahi** [ALIYA2Group/Mod20_Segment_2#15] [Ideas]
We need atleast 1,000 data points. I am doing the ML and Database based on existing data for now  (199 rows each for north and south (south_data_finalized_1 and north_data_finalized_1)

So my suggestion is to use data daily data from 2003 - this will include  xco2 and xch4 data - which will require redoing the extraction of larger data set of the other variables but it would give us the required dataset on a daily basis from 2003 to 30th june 2020 - a total of 6061 data points

**Comment by amberwnaushahi** [comment]
Also, please add a config.py file in your Database folder with your PGAdmin password as db_password = 'xxxxxx' to ensure you can access the database on your PCs

**Reply by 735713038455163** [reply]
this was resolved

**Comment by amberwnaushahi** [comment]
In the class, I want to ask TAs at what stage should we be creating database (I think it should be within the ETL Files and not separately)

Source Dataset ---> ETL and joining tables ---> load into Postgres ---> read into ML

**Reply by 735713038455163** [reply]
The me know if mod there this is still concern @amberwnaushahi
